webpack.config.js

```diff
@@ -44,20 +78,20 @@ module.exports = {
     theme_oceans: './assets/src/styles/theme_oceans.scss',
     theme_oil: './assets/src/styles/theme_oil.scss',
     theme_plastic: './assets/src/styles/theme_plastic.scss',
-    ...entryPoints('Accordion'),
-    ...entryPoints('Covers'),
-    ...entryPoints('CarouselHeader'),
-    ...entryPoints('Spreadsheet'),
-    ...entryPoints('Timeline'),
+    AccordionStyle: './assets/src/styles/blocks/Accordion/AccordionStyle.scss',
```

Comment:
You can do sth like this and you'll avoid repeating code

```
{
  entry: {
    theme_oceans: './assets/src/styles/theme_oceans.scss',
    theme_oil: './assets/src/styles/theme_oil.scss',
    theme_plastic: './assets/src/styles/theme_plastic.scss',
    ...['AccordionStyle', 'SpreadsheetStyle'].reduce((b,c) => {
      if(c) {
        return {
          ...b,
          [c]: `./assets/src/styles/blocks/${c}/${c}Style.scss`,
        };
      }
    }, []),
  },
}
```

Reply:
For a webpack config I'd rather keep it simple. A reducer would make the file significantly harder to read for newcomers.

Reply:
Btw, why is it a reducer and not just a map? :thinking: `if (c)` will always be true for an array of non empty string literals

Reply:
Yes, It's a good point. We can also use `map`, however, take in mind human errors 👀 if you use an Array 😄